openapi/workflow_execution_service.openapi.yaml

```diff
@@ -664,7 +684,7 @@ components:
         runs:
           type: array
           items:
-            $ref: '#/components/schemas/State'
+            $ref: '#/components/schemas/RunListStatus'
```

Comment:
Actually, I'm a bit confused, as I never knew we had a version of the specs that _only_ returned the `State`.

In our WES implementation we return `RunStatus` which gives you the `run_id` of the run, together with the state. 

Considering that both [`v1.0.0`](https://github.com/ga4gh/workflow-execution-service-schemas/releases/tag/1.0.0) and [`v1.0.1`](https://github.com/ga4gh/workflow-execution-service-schemas/releases/tag/1.0.1) still basically have the schema `RunListResponse` schema I describe (showing here for the OpenAPI v3 version in `v1.0.1`]):

```yaml
    RunListResponse:
      title: RunListResponse
      type: object
      properties:
        runs:
          type: array
          items:
            $ref: '#/components/schemas/RunStatus'
          description: A list of workflow runs that the service has executed or is executing. The list is filtered to only include runs that the caller has permission to see.
        next_page_token:
          type: string
          description: A token which may be supplied as `page_token` in workflow run list request to get the next page of results.  An empty string indicates there are no more items to return.
      description: The service will return a RunListResponse when receiving a successful RunListRequest.
```

it seems to me that somewhere in `develop` a breaking change was introduced that changed `RunStatus` to `State` and causes this issue. This was possibly an accident, because just returning a list of states is really not very useful. It is also not extensible, thus causing us to introduce a breaking change here again.

So maybe let's have a PR first for fixing this regression (at least that's how I see it) before tagging the next release after `v1.0.1` (so far it only seems to be on the `develop` branch, as the head commit on the `master` branch also returns `RunStatus` items, see [here](https://github.com/ga4gh/workflow-execution-service-schemas/blob/13f56821fb50e4b01a269354421c96c20785e909/openapi/workflow_execution_service.openapi.yaml#L524-L536)), then merge that in this branch to avoid a breaking change here.

Reply:
I have just checked and the commit that (essentially) changed the items returned via `RunListResponse` from `RunStatus` to `State` is this one: https://github.com/ga4gh/workflow-execution-service-schemas/commit/ddb98d7d8479fed209e84113f85ee2023e033026

I don't think it's been tagged or released yet, so I think we can just change this back. I would do it in a separate PR though to make it clear that this reverts a breaking change that was inadvertently introduced. This will then also make it clear that _this_ PR is non-breaking.

Reply:
@uniqueg thanks for pointing this out. Only requiring state is definitely a bug that wormed its way in during the major refactor of the openapi docs. I will fix that in this PR.

As per the difference between RunListStatus and RunStatus, I agree it does not make much sense to separate them currently. I think what is true of listing (wanting a bit extra info) should also be true of the `/status` endpoint. I will remove the new entity and add the changes to `RunStatus` instead

Reply:
Great.

Make sure to point out in the PR description that it also reverses an inadvertently introduced breaking change.